integrations/quick-start-guide.md

```diff
@@ -0,0 +1,48 @@
+# Integration Management Quickstart Guide
+In this quickstart guide, we will go over key features for integration management.
+
+## Integrations Page
+The integrations page is your home for adding and viewing integrations connected to your Aqueduct account. You can naviate to integrations page by clicking `Integrations` in the left menu bar. This page has two main sections: [add an integration](#add-an-integration) and [connected integrations](#connected-integrations).
+
+### Add an Integration
+To add an integration, simply click the icon for the integration you would like to connect. You would need to provide your service details together with your credentials. You could find more details [here](./adding-an-integration/) on connecting to s3, bigquery, and Google Cloud Storage (GCS).
+
+Once you added an integration, you can refer to [using integrations guide](./using-integrations/) to use the integration in Aqueduct SDK.
+
+### Connected Integrations
+You can view a list of all connected integrations. You can click into each item to view more details or manage that integration. Every Aqueduct server comes with a [demo integration](./aqueduct-demo-integration.md) that you can play around with.
+
+## Integration Detail Page
+Integration Detail Page is the console for you to manage a single integration. You can navigate to this page by clicking an item in [Connected Integrations](#connected-integrations) from [Integrations Page](#integrations-page) you can view details about an integration and make changes to it:
+* [Integration Information](#integration-information)
+* [Test-Connecting an Integration](#test-connecting-an-integration)
+* [Editing an Integration](#editing-an-integration)
+* [Deleting an Integration](#deleting-an-integration)
+* [Viewing Data in an Integration](#viewing-data-in-an-integration)
+* [Viewing Workflows Using an Integration](#viewing-workflows-using-an-integration)
+
+### Integration Information
+At top of the integration detail page. You can view basic information about this integration. There are also a number of options to manage it, including [test-connection](#test-connecting-an-integration), [edit the integration](#editing-an-integration), and [delete the integration](#deleting-an-integration).
+![](<../.gitbook/assets/integration_information.png>)
+
+### Test-Connecting an Integration
+In `Options` menu, you can test-connect an integration to verify if it's currently available. This is especially helpful to confirm whether your credentials are expired.
```

Comment:
In *the* `Options` menu

Reply:
"test-connect" isn't a word, I would say "you can test Aqueduct's connection to your integration"